PyInstaller/building/utils.py

```diff
@@ -473,3 +475,93 @@ def format_binaries_and_datas(binaries_or_datas, workingdir=None):
                                 os.path.normpath(src_file)))
 
     return toc_datas
+
+
+def _load_code(modname, filename):
```

Comment:
This looks duplicate to what modulegraph already does. Why do we need this here?